spec/models/pieces/king_spec.rb

```diff
@@ -1,4 +1,45 @@
 require 'rails_helper'
 
 RSpec.describe King, type: :model do
+  before :all do
+    @game = FactoryGirl.create(:game)
+    Piece.destroy_all
+    @king = FactoryGirl.create(:king)
+  end
```

Comment:
I have an rspec trick for you :smile:.

Another way of writing this is by using `let` in Rspec.

`let(:king) { FactoryGirl.create(:king) }`

Then in all your tests, you can use `king` instead of `@king`

For more info: http://betterspecs.org/#let (+ they include reasons why `let` is better than using instance variables)